src/index.ts

```diff
@@ -1 +1,3 @@
-export * from './components/buttons/DefaultButton/DefaultButton';
\ No newline at end of file
+export * from './components/atoms/buttons/DefaultButton/DefaultButton';
+export * from './components/atoms/input/TextInput/TextInput';
```

Comment:
just putting the comment about pulling out the testentry stuff here so it gets done before we can merge the code